content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,30 @@
+## Cascade Learning {.page_break_before}
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and the predictions there-of to make an educated decision about the identified predictive patterns, collectively called _ensemble learning_. 
+In an ensemble learning workflow, one may use various parameters to maximize the learned information. 
```

Comment:
I'm not sure what this sentence does for the intended audience.

Reply:
This was to segway into the next sentence where we differentiate between ensembles-using-consensus vs ensembles-using-cascades

Reply:
Gotcha – in that case, I think it needs more context to connect the two.